commons-statistics-regression/README.md

```diff
@@ -0,0 +1,105 @@
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<!---
+ +======================================================================+
+ |****                                                              ****|
+ |****      THIS FILE IS GENERATED BY THE COMMONS BUILD PLUGIN      ****|
+ |****                    DO NOT EDIT DIRECTLY                      ****|
+ |****                                                              ****|
+ +======================================================================+
+ | TEMPLATE FILE: readme-md-template.md                                 |
+ | commons-build-plugin/trunk/src/main/resources/commons-xdoc-templates |
+ +======================================================================+
+ |                                                                      |
+ | 1) Re-generate using: mvn commons:readme-md                          |
+ |                                                                      |
+ | 2) Set the following properties in the component's pom:              |
+ |    - commons.componentid (required, alphabetic, lower case)          |
+ |    - commons.release.version (required)                              |
+ |                                                                      |
+ | 3) Example Properties                                                |
+ |                                                                      |
+ |  <properties>                                                        |
+ |    <commons.componentid>math</commons.componentid>                   |
+ |    <commons.release.version>1.2</commons.release.version>            |
+ |  </properties>                                                       |
+ |                                                                      |
+ +======================================================================+
+--->
+Apache Commons Statistics Regression
+===================
+
+[![Build Status](https://travis-ci.org/apache/commons-statistics-regression.svg?branch=master)](https://travis-ci.org/apache/commons-statistics-regression)
+[![Coverage Status](https://coveralls.io/repos/apache/commons-statistics-regression/badge.svg?branch=master)](https://coveralls.io/r/apache/commons-statistics-regression)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.apache.commons/commons-statistics-regression/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.apache.commons/commons-statistics-regression/)
+[![License](http://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
+
+Statistical regressions.
+
+Documentation
+-------------
+
+More information can be found on the [Apache Commons Statistics Regression homepage](https://commons.apache.org/proper/commons-statistics).
```

Comment:
It looks like this should just say "Apache Commons Statistics Homepage"?

Reply:
I have changed it in distribution and regression now.